internal/bminventory/inventory.go

```diff
@@ -559,7 +543,22 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		TriggerMonitorTimestamp:     time.Now(),
 		MachineNetworkCidrUpdatedAt: time.Now(),
 	}
+	if params.NewClusterParams.OlmOperators != nil {
+		var newOLMOperators []*models.MonitoredOperator
+		newOLMOperators, err = b.getOLMOperators(&cluster, params.NewClusterParams.OlmOperators)
+		if err != nil {
+			return nil, err
+		}
 
+		err = operators.EnsureLVMAndCNVDoNotClash(*releaseImage.Version, newOLMOperators)
```

Comment:
is this guaranteed to be populated? not sure what is `releaseImage`

Reply:
this part of the code is already there, it is just moved down, also I think it should be populated always

Reply:
this part is required to move since `b.getOLMOperator` requires `cluster` as a parameter which will not be available if we move it above, Also`releaseImage` is already used in `Cluster: models.Cluster`.